bindata/network/ovn-kubernetes/006-ovs-node.yaml

```diff
@@ -87,6 +90,25 @@ spec:
           name: var-lib-openvswitch
         - mountPath: /env
           name: env-overrides
+        - mountPath: /run/systemd
+          name: run-systemd
+        - mountPath: /sysroot
+          name: host-sysroot
+        - mountPath: /var/log/openvswitch
+          name: var-log-openvswitch
+        - mountPath: /usr/shre/openvswitch/scripts
```

Comment:
@mccv1r0 /usr/share? Looks like we don't actually use it since we're using the container's scripts here.